contracts/dtoken/src/liquidation.rs

```diff
@@ -0,0 +1,45 @@
+use crate::*;
+
+#[near_bindgen]
+impl Contract {
+    pub fn liquidation(
```

Comment:
@convowork1 this function should return `PromiseOrValue` as it's only part of the Liquidation flow